### PR TITLE
add iovlen function so builds on ARM32 work

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

**Comment by rminnich** [comment]
yeah, that's going to take longer, I no longer understand the u-root circleci setup.

**Comment by hugelgupf** [comment]
This is p9 though -- look at .CircleCI/config.yml. it'll be super easy

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,7 +5,19 @@ jobs:
...
+      - run: GOARCH=amd64 go build ./...
+      - run: GOARCH=arm go build ./...
+      - run: GOARCH=arm64 go build ./...
+      - run: GOARCH=mips go build ./...
+      - run: GOARCH=mips64 go build ./...
+      - run: GOARCH=mips64le go build ./...
+      - run: GOARCH=mipsle go build ./...
+      - run: GOARCH=ppc64 go build ./...
+      - run: GOARCH=ppc64le go build ./...
+      - run: GOARCH=s390x go build ./...
```
Comment (hugelgupf) [inline .circleci/config.yml]: Nice. That's awesome.

**Comment by rminnich** [comment]
it's not pretty but linux doesn't make it simple. 

bluecmd can now run this on his mainframe.

**Pull request merged**